GRDB/QueryInterface/Schema/VirtualTableModule.swift

```diff
@@ -66,7 +75,17 @@ extension Database {
         let sql = chunks.joined(separator: " ")
         try execute(sql: sql)
     }
-    
+
+    private func makeTableDefinition<Module: VirtualTableModule>(_ module: Module, _ metadata: ModuleMetadata) -> Module.TableDefinition
+    {
+        module.makeTableDefinition()
+    }
```

Comment:
Problem: this method is always called. Not sure how to have the other one used instead.

Reply:
You're right.

I have another idea.